files/init-container.sh

```diff
@@ -17,6 +17,9 @@ case $GIT_BRANCH in
 	falcon)
 		XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS9mYTdjMGVhOS05ZDMxLTUwYmItYThkNi04YWUzNjdlZjJmMTQvZG9tYWluMA==
 		;;
+  vgpu-migration)
+    XS_REPO_KEY=aHR0cDovL3hhcGktcmVwb3MuczMtd2Vic2l0ZS11cy1lYXN0LTEuYW1hem9uYXdzLmNvbS9mZWE3NTJlNzc5MjUzMTkyYTcxMDIzMmFmNDNlYWJjZC9kb21haW4w
```

Comment:
This key needs to have the dashes in the UUID part (because the upload script keeps the dashes when uploading; see https://github.com/xenserver/upload-scripts/blob/master/src/update_xs_yum.ml#L497).